routing/routing.xml

```diff
@@ -614,12 +630,22 @@
 			<select value="1.3" t="highway" v="steps"/>
 			<select value="3" t="route"   v="ferry"/>
 			<select value="5" t="bicycle" v="dismount"/>
-			<select value="22" t="bicycle" v="designated"/>
-			<if param="avoid_unpaved">
-				<select value="4" t="highway" v="track"/>
-				<select value="4" t="highway" v="footway"/>
-				<select value="3" t="highway" v="path"/>
-			</if>
+			<!-- <select value="22" t="bicycle" v="designated"/>-->
+			<!-- Line above: Does the "bicycle designated" work here?
+			So far, we have assigned a kmh value to tracks, paths etc - 
+			can a kmh value also be assigned to a tag like "bicycle designated"?
+			If so: Would all tracks, paths etc tagged with "bicycle designated"
+			be multiplied later on (so that, if they are tagged with "paved" or
+			"asphalt", the 23 kmh would be multiplied with the "paved" factor)?
+			Could someone please help me understand this?
+			If it works as described, it adds some great possibilities.-->
```

Comment:
Thats a good idea. I did not think of that. So when we put it here, it will just get a fixed value, without any other multiplication? 

In that case, we should even set it even higher, maybe 35 kmh. Because the multiplications can lead to very high numbers as well (for example a track with bicycle=official or a residential road with a cycle track).